doc/source/User_guide/extensions.rst

```diff
@@ -0,0 +1,36 @@
+Extensions
+==========
+
+PyAEDT extensions provide a simplified interface to perform automated workflows.
+The user can install or uninstall these extensions using the extension manager.
```

Comment:
```suggestion
Extensions can be added or removed from AEDT using the extension manager.
```

Reply:
It should be active voice and second person:
In AEDT, you can use the Extension Manager to install or uninstall extensions.